src/CellData/DiracDeltas.jl

```diff
@@ -65,14 +70,63 @@ function DiracDelta{0}(model::DiscreteModel;tags)
   DiracDelta{0}(model,degree;tags=tags)
 end
 
-function (d::DiracDelta)(f)
+function (d::GenericDiracDelta)(f)
   evaluate(d,f)
 end
 
-function evaluate!(cache,d::DiracDelta,f)
+function evaluate!(cache,d::GenericDiracDelta,f)
   ∫(f)*d.dΓ
 end
 
-function get_triangulation(d::DiracDelta)
+function get_triangulation(d::GenericDiracDelta)
   d.Γ
 end
+
+# For handling DiracDelta at a generic Point in the domain #
+
+function _cell_to_pindices(pvec::Vector{<:Point},trian::Triangulation)
+  cache = _point_to_cell_cache(KDTreeSearch(),trian)
+  cell_to_pindex = Dict{Int, Vector{Int32}}()
+  for i in 1:length(pvec)
+    cell = _point_to_cell!(cache, pvec[i])
+    push!(get!(() -> valtype(cell_to_pindex)[], cell_to_pindex, cell), i)
+  end
+  cell_to_pindex
+end
+
+function DiracDelta(model::DiscreteModel{D}, p::Point{D,T}) where {D,T}
+  trian = Triangulation(model)
+  cache = _point_to_cell_cache(KDTreeSearch(),trian)
+  cell = _point_to_cell!(cache, p)
+  trianv = TriangulationView(trian,[cell])
+  pquad = GenericQuadrature([p],[one(T)])
+  pmeas = Measure(CellQuadrature([pquad],[[p]],[[one(T)]],trianv,PhysicalDomain(),PhysicalDomain()))
```

Comment:
to be on the safe side w.r.t. memory allocations, better not to create `[p]` and `[one(T)]` twice.

why do we have to pass `[[p]]` to `CellQuadrature` constructor? Couldn't it get this info from `pquad` ? (perhaps a missing constructor in Gridap?)

Reply:
Yes true, I was thinking the same, may be a missing constructor, specifically for `data_domain_style` being `PhysicalDomain()`, the problem is we can't use the `compress_cell_data` for `AbstractArrays` only for `Fill` and `CompressedArray`, which is the usual quadrature case. Yes I ll remove the repetition and reuse.

Reply:
> Yes true, I was thinking the same, may be a missing constructor, specifically for data_domain_style being PhysicalDomain(), the problem is we can't use the compress_cell_data for AbstractArrays only for Fill and CompressedArray, which is the usual quadrature case. Yes I ll remove the repetition and reuse.

Ok, I see. The constructor is missing, but if we added it, we would not be able to leverage it in our particular context (we do not have Fill or CompressedArrays to represent the quadrature points and weights).